starter-code/css/main.css

```diff
@@ -0,0 +1,113 @@
+body{
+    font-family: Lato, sans-serif;
+    padding-left: 30px;
+    margin-left: 20px;
+    border-left-style: solid;
+    border-left-color: rgba(195,195,195,.8);
+
+}
+
+header{
+  padding-top: 40px;
+}
+
+a{
+  text-decoration: none;
+  all: unset;
```

Comment:
Rather than using this CSS property, you will want to consider a normalize or reset css sheet instead.